src/authentication/team-token.ts

```diff
@@ -0,0 +1,58 @@
+import * as Knex from 'knex';
+import * as moment from 'moment';
+
+const randomstring = require('randomstring');
+
+export const teamTokenLength = 7;
```

Comment:
I'd make this much bigger, on the order of 16 chars. Just to be on the safe side.